examples/conversation_sql/conversation_sql.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+"""A simple example of agent that can perform
+SQL queries through natural language conversation.
+"""
+from sql_utils import (
+    DailSQLPromptGenerator,
+    execute_query,
```

Comment:
plz use `query_sqlite` in agentscope instead of `execute_query`

Reply:
Done